dot/config.go

```diff
@@ -185,64 +184,66 @@ func (p PprofConfig) String() string {
 	return p.Settings.String()
 }
 
-// GssmrConfig returns a new test configuration using the provided basepath
-func GssmrConfig() *Config {
+// WestendDevConfig returns a kusama node configuration
+func WestendDevConfig() *Config {
 	return &Config{
 		Global: GlobalConfig{
-			Name:           gssmr.DefaultName,
-			ID:             gssmr.DefaultID,
-			BasePath:       gssmr.DefaultBasePath,
-			LogLvl:         gssmr.DefaultLvl,
-			MetricsAddress: gssmr.DefaultMetricsAddress,
-			RetainBlocks:   gssmr.DefaultRetainBlocks,
-			Pruning:        pruner.Mode(gssmr.DefaultPruningMode),
-			TelemetryURLs:  gssmr.DefaultTelemetryURLs,
+			Name:           "Westend",
+			ID:             "westend_dev",
+			BasePath:       "~/.gossamer/westend-dev",
+			LogLvl:         log.Info,
+			MetricsAddress: ":9876",
```

Comment:
Should we only listen on localhost with

```suggestion
			MetricsAddress: "localhost:9876",
```

Reply:
tagging Tim for this one @timwu20 what you think?

Reply:
I think it should listen on all addresses.  It's not understood that metrics are only accessible locally.